Sphinx not rendering bullets from RST correctly

For some reason - and I don't think it is simply getting the RST syntax wrong because I've tried all combinations under the sun - `sphinx` is refusing to render bulleted lists correctly. The list items just come out as a set of paragraphs in the HTML.

_Originally posted by @davidorme in https://github.com/ImperialCollegeLondon/virtual_rainforest/pull/122#discussion_r1082659648_

I'm sorry if it is just me being dumb, but I'm trying to build the documentation locally (with `make html` in the `docs` directory) to diagnose this and I end up with errors that build down to:

```error
[ERROR] - manager - wrapper(89) - No such kernel named vr_python3
```

I did `git pull` in the develop branch  followed by `poetry install` to get the latest dependencies (with the versions indicated in the lock file) without success.

Am I missing anything here?

--- a/.readthedocs.yaml
+++ b/.readthedocs.yaml
@@ -17,6 +17,7 @@ build:
       - pip install poetry
       # Tell poetry to not use a virtual environment
       - poetry config virtualenvs.create false
+    post_install:
       # Install dependencies with 'docs' dependency group
       # https://python-poetry.org/docs/managing-dependencies/#dependency-groups
       - poetry install --with docs

